src/css/var/isHiddenWithinTree.js

```diff
@@ -12,7 +12,12 @@ define( [
 		// isHiddenWithinTree might be called from jQuery#filter function;
 		// in that case, element will be second argument
 		elem = el || elem;
-		return jQuery.css( elem, "display" ) === "none" ||
-			!jQuery.contains( elem.ownerDocument, elem );
+		return elem.style.display === "none" ||
```

Comment:
I would add a bunch of comments which would explain every line, logic is pretty bizarre for unprepared eyes